ptbcontrib/username_to_chat_api/usernametochatapi.py

```diff
@@ -20,63 +20,51 @@
 This module contains a class that, once initiated, works as a shortcut to the UsernameToChatAPI
 and puts the response in a Chat object, as well as puts the error to the fitting TelegramErrors.
 """
+from httpx import AsyncClient
 from telegram import Bot, Chat, error
 
-try:
-    from telegram.vendor.ptb_urllib3 import urllib3
-except ImportError:  # pragma: no cover
-    import urllib3
 
-try:
-    import ujson as json
-except ImportError:  # pragma: no cover
-    import json  # type: ignore[no-redef]
-
-
-class UsernameToChatAPI:  # pylint: disable=too-few-public-methods
+class UsernameToChatAPI:
     """
-    This class stores the URL and api key for a UsernameToChatAPI instance, and then provides a
-    method to get the Chat object for an username.
-
-     Args:
-         api_url (:obj:`str`): The base URL (speak: domain) to the API.
-         api_key (:obj:`str`): The key, necessary to access the API.
-         bot (:obj:`telegram.Bot`): An initiated bot object, to create the Chat object.
-         pool_manager (:obj:`urllib3.PoolManager`, optional): Pre initialized
-             :obj:`urllib3.PoolManager`.
+    Args:
+        api_url (:obj:`str`): URL to the API instance.
+        api_key (:obj:`str`): Key for the API.
+        bot (:class:`telegram.Bot`): Bot instance, used to create the Chat object.
```

Comment:
does this have to be initialized? In the readme example, you call neither `bot.initialize` nor `bot.shutdown` and also don't use a context manager 🤔

Reply:
What would be the correct type hint for a not initialized class? As long as de_json doesn't care, I don't care about Bot being initialized or not, as you probably guessed from the code.

Reply:
there is no type hint for that. had in fact not noticed that it's only used in de_json 😬 should be okay, then